packages/css-jss/src/types.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+export type ClassName = string
+
+export type Style = {} | null | void | '' | ClassName
```

Comment:
Using ’ClassName’ here is a bit confusing here. Is it really a class name or a css string?

Reply:
well it corresponds the className prop

Reply:
actually using it to be more explicit here about the string

Reply:
but actually it doesn't matter that much, moving it to just string